index.js

```diff
@@ -1,5 +1,10 @@
 /**
  * @typedef {import('hast').Root} Root
+ * @typedef {string} Prefix
+ * @typedef Options
+ *   Configuration.
+ * @property {Prefix} [prefix='']
+ * 
  */
```

Comment:
```suggestion
 * @typedef {import('hast').Root} Root
 */

/**
 * @typedef Options
 *   Configuration (optional).
 * @property {string} [prefix='']
 *   Prefix all IDs with this value.
 */
```

Adds docs, removes unneeded alias, splits “imports” from “definitions”